app/main.py

```diff
@@ -1,6 +1,22 @@
-# write your imports here
+from app.people.customer import Customer
+from app.people.cinema_staff import Cleaner
+from app.cinema.bar import CinemaBar
+from app.cinema.hall import CinemaHall
 
 
-def cinema_visit(customers: list, hall_number: int, cleaner: str, movie: str):
-    # write you code here
-    pass
+def cinema_visit(customers: list,
+                 hall_number: int,
+                 cleaner: str,
+                 movie: str
+                 ) -> None:
+    CinemaBar()
+    for customer in customers:
+        CinemaBar.sell_product(
+            Customer(customer["name"],
+                     customer["food"]),
+            customer["food"])
+    CinemaHall.movie_session(
+        CinemaHall(hall_number), movie,
```

Comment:
```suggestion
    CinemaHall(hall_number).movie_session(
       movie,
```
Call method to instance!!!!